pkg/auth/artifact_verifier_test.go

```diff
@@ -45,7 +46,13 @@ func testVerifiedWithFiles(t *testing.T, files []testFile, verifier ArtifactVeri
 	if err != nil {
 		t.Fatal(err)
 	}
-	err = verifier.VerifyHoistArtifact(localCopy, fmt.Sprintf("file://%v", filePath))
+
+	url, err := url.Parse(fmt.Sprintf("file://%v", filePath))
```

Comment:
Instead of constructing a string and parsing it, can you just create a `*url.URL` object yourself?